backend/src/openarchiefbeheer/destruction/utils.py

```diff
@@ -317,3 +319,37 @@ def prepopulate_selection_after_review_response(
         )
 
     SelectionItem.objects.bulk_create(other_selection_items_to_create)
+
+
+def get_destruction_report_url(destruction_list: "DestructionList") -> str | None:
```

Comment:
Wouldn't it make sense to have this on the model?

Reply:
Yes! Good idea